pkg/crcontrollers/reservation/reservationcontroller.go

```diff
@@ -86,14 +87,18 @@ func (c *Controller) handleReservationUpdate(ctx context.Context, log *logrus.En
 		}
 
 		// TODO: do not read all ACs and ACRs for each request: https://github.com/dell/csi-baremetal/issues/89
-		acReader := capacityplanner.NewACReader(c.client, c.log, true)
-		acrReader := capacityplanner.NewACRReader(c.client, c.log, true)
-		reservedCapReader := capacityplanner.NewUnreservedACReader(c.log, acReader, acrReader)
-		capManager := c.capacityManagerBuilder.GetCapacityManager(c.log, reservedCapReader)
+		acReader := capacityplanner.NewACReader(c.client, log, true)
+		acrReader := capacityplanner.NewACRReader(c.client, log, true)
+		capManager := c.capacityManagerBuilder.GetCapacityManager(log, acReader, acrReader)
 
 		requestedNodes := reservationSpec.NodeRequests.Requested
 		placingPlan, err := capManager.PlanVolumesPlacing(ctx, volumes, requestedNodes)
+		if err == baseerr.ErrorAnotherACRReserved {
+			log.Warningf("Consistent LVG volumes reservation feature is enabled: %s", err.Error())
```

Comment:
Please update message to `Reservation request rejected due to ...`

Reply:
updated